operate/common/src/main/java/io/camunda/operate/property/WebSecurityProperties.java

```diff
@@ -72,8 +89,7 @@ public boolean getHttpStrictTransportSecurityIncludeSubDomains() {
 
   public WebSecurityProperties setHttpStrictTransPortSecurityIncludeSubDomains(
       final boolean httpStrictTransPortSecurityIncludeSubDomains) {
-    this.httpStrictTransportSecurityIncludeSubDomains =
-        httpStrictTransPortSecurityIncludeSubDomains;
+    httpStrictTransportSecurityIncludeSubDomains = httpStrictTransPortSecurityIncludeSubDomains;
```

Comment:
What is the value of the field after this expression?

Reply:
This sets the class variable to be equal to the input parameter. My IDE automatically removed it as it deems it unnecessary to use `this`